README.md

```diff
@@ -99,32 +99,73 @@ Output formats are:
 
 #### ifs
 Ifs output format print all values of a list or a single value in one line.
- All values are separated by the content of IFS environment variable if defined,
-space otherwise. This is usefull in a shell 
-for loop, but your content must, of course, don't contain IFS value.
-```sh
+All values are separated by the content of IFS environment variable if defined,
+space otherwise.
+
+Examples (consider the previous [YAML file example](#with-yaml-file)):
+```shell
+$ IFS="|" niet /path/to/your/file.yaml .project.list-items -f ifs
+item1|item2|item3
+$ IFS=" " niet /path/to/your/file.yaml .project.list-items -f ifs
+item1 item2 item3
+$ IFS="@" niet /path/to/your/file.yaml .project.list-items -f ifs
+item1@item2@item3
+```
+
+This is usefull in a shell for loop,
+but your content must, of course, don't contain IFS value:
+```shell
 OIFS="$IFS"
 IFS="|"
-for i in $(niet tests/samples/sample.yaml .project.list-items -f ifs); do
-	echo $i
+for i in $(niet /path/to/your/file.yaml .project.list-items -f ifs); do
```

Comment:
Why don't you want to use our samples files ? I used this one to allow to download this file et perform the checks on it. (Maybe we could add the dl URL in this doc for all samples to allow users to test on same files)

Reply:
In accordance to the previous examples in the documentation I prefer to use it along the documentation.